### PR TITLE
Fixed wait bug when shrinking

### PR DESCRIPTION
**Pull request by ggrieco-tob**
#582

### CODE REVIEW

**Comment by agroce** [comment]
(As noted, this fixes the problem for me, with proper shrinking output)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -15,12 +15,12 @@ jobs:
       matrix:
         os:
           - ubuntu-latest
-          - macos-latest
+            #- macos-latest
```
Comment (arcz) [inline .github/workflows/ci.yml:18]: is this intended in this PR?

**Review by arcz — DISMISSED** [review]
This looks fine, just the ci.yml to check

File: .github/workflows/ci.yml (hunk 1 above)
Comment (ggrieco-tob) [reply .github/workflows/ci.yml:18]: yes, macos building was failing. I think we should disabled it until we can make it work again (or remove it)

**Review by incertia — APPROVED** [review]

**Pull request merged**